system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -186,6 +186,9 @@ match_types! {
 	pub type FellowshipSalaryPallet: impl Contains<MultiLocation> = {
 		MultiLocation { parents: 1, interior: X2(Parachain(1001), PalletInstance(64)) }
 	};
+	pub type AmbassadorSalaryPallet: impl Contains<MultiLocation> = {
```

Comment:
I had a test for this, but for now there is no crate to place it, it is work in process.

test: https://github.com/paritytech/polkadot-sdk/blob/43fceb5f4eda92d34ee35ceb0b2addb382704f75/cumulus/parachains/integration-tests/emulated/collectives/collectives-polkadot/src/tests/ambassador.rs#L25 
from this PR: https://github.com/paritytech/polkadot-sdk/pull/1308